clang/test/dpct/thrust-algo.cu

```diff
@@ -26,6 +26,23 @@ void k() {
   auto bo = [](int x, int y) -> int { return x + y; };
   auto gen = []() -> int { return 23; };
 
+  thrust::maximum<int> binary_op;
+
+  // exclusive_scan
+
+  // CHECK: std::exclusive_scan(oneapi::dpl::execution::seq, v.begin(), v.end(), v2.begin(), 0);
+  thrust::exclusive_scan(thrust::host, v.begin(), v.end(), v2.begin());
+  // CHECK: std::exclusive_scan(oneapi::dpl::execution::make_device_policy(q_ct1), v.begin(), v.end(), v2.begin(), 0);
+  thrust::exclusive_scan(v.begin(), v.end(), v2.begin());
```

Comment:
As v.begin(), v.end(), v2.begin() are host iterator, we should use "oneapi::dpl::execution::seq" in the migrated code. 
if v.begin(), v.end(), v2.begin() are device iterator, we should use "oneapi::dpl::execution::make_device_policy(q_ct1)" in the migrated code.
You can refer to Line244, Line245.